src/gate/printer/GraphML.cpp

```diff
@@ -0,0 +1,49 @@
+#include "gate/model/gnet.h"
```

Comment:
Where are header lines of the project?
Also, please add a header file for the graphml.cpp.
Also, please use a lower register in the file names.
Also, please add the code into the project that is switch on a process of your file compilation.
Where is a test for the developed functionality?